test/config/path-util.js

```diff
@@ -0,0 +1,155 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+const expect = require('chai').expect;
+const WebpackConfig = require('../../lib/WebpackConfig');
+const RuntimeConfig = require('../../lib/config/RuntimeConfig');
+const pathUtil = require('../../lib/config/path-util');
+const process = require('process');
+
+function createConfig() {
+    const runtimeConfig = new RuntimeConfig();
+    runtimeConfig.context = __dirname;
+    runtimeConfig.environment = 'dev';
+    runtimeConfig.babelRcFileExists = false;
+
+    return new WebpackConfig(runtimeConfig);
+}
+
+/**
+ * Some tests are very specific to different operating systems.
+ * We use this to only run them when needed.
+ *
+ * @returns {boolean}
+ */
+function isWindows() {
```

Comment:
this could be a const rather than a function, as the value being returned will always be the same:

```js
const isWindows = process.platform === 'win32'

// ...

        if (isWindows) {
            this.skip();
        }
```